### ISSUE TITLE
black screen in Every Start up

### ISSUE THREAD

**Issue by hhlp** [flathub/com.axosoft.GitKraken#147]
System Linux: x64
Dist.: Fedora 36

flatpak version:

flatpak-builder-1.2.2-2.fc36.x86_64
flatpak-session-helper-1.12.7-2.fc36.x86_64
flatpak-selinux-1.12.7-2.fc36.noarch
flatpak-1.12.7-2.fc36.x86_64
flatpak-libs-1.12.7-2.fc36.x86_64

Graphics:  Device-1: NVIDIA GK208B [GeForce GT 730] vendor: Micro-Star MSI driver: nouveau v: kernel
           bus-ID: 2d:00.0 chip-ID: 10de:1287 class-ID: 0300
           Device-2: Logitech HD Webcam C525 type: USB driver: snd-usb-audio,uvcvideo bus-ID: 1-2:3
           chip-ID: 046d:0826 class-ID: 0e02 serial: <filter>
           Display: wayland server: X.Org 1.21.1.4 compositor: gnome-shell driver: loaded: nouveau
           note: n/a (using device driver) - try sudo/root resolution: 1920x1080~75Hz s-dpi: 96
           OpenGL: renderer: NV106 v: 4.3 Mesa 21.3.3 direct render: Yes

![Captura de pantalla de 2022-06-26 08-20-01](https://user-images.githubusercontent.com/2659606/175802203-72f03878-b491-4809-a71c-91605bc0e599.png)

Only work with this Parameter Enable in previous version of flatpak for example Fedora 35, in fedora 36 this parameters is missing:

```
flatpak run --disable-gpu .......
```

I've had working in previous version of fedora 35 with noveau Dirvers....

Regards.,

**Comment by Lctrs** [comment]
Have you tweaked the sandbox permissions ?

**Comment by hhlp** [comment]
No., I've  just install and use it., I didn't tweak any configuration

Regards.,

**Comment by hhlp** [comment]
I've also have this error:

![Captura desde 2022-08-17 08-48-19](https://user-images.githubusercontent.com/2659606/185053600-4f970ed0-05f8-4008-9588-bdcb920291dd.png)

**Comment by Eonfge** [comment]
Please, re-open if this problem is still relevant.
